cmd/kops/create_cluster.go

```diff
@@ -1008,6 +1065,7 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		}
 	}
 
+	// Can we acutally get to this if??
```

Comment:
Do you mean that targetName is always `!= ""` ?

Reply:
Is targetName ever == "".  I think it is always set, and the if statement is not needed

Reply:
I think you're right, but separate PR probably

Reply:
Can you pull this comment out of this PR?